downstream/modules/platform/proc-controller-credential-create-openshift-account.adoc

```diff
@@ -8,8 +8,49 @@ After you create the service account, its credentials are provided to {Controlle
 After you create a service account, use the information in the new service account to configure {ControllerName}.
 
 .Procedure
-. To create a service account, download and use the link:https://docs.ansible.com/automation-controller/latest/html/userguide/_downloads/7a0708e6c2113e9601bf252270fa6c50/containergroup-sa.yml[sample service account] and change it as required to obtain the previous credentials.
-. Apply the configuration from the link:https://docs.ansible.com/automation-controller/latest/html/userguide/_downloads/7a0708e6c2113e9601bf252270fa6c50/containergroup-sa.yml[sample service account]:
+. To create a service account, download and use the sample service account, `containergroup sa` and change it as needed to obtain the credentials:
```

Comment:
I would recommend adding a comma after `containergroup sa`: 

"To create a service account, download and use the sample service account, `containergroup sa`, and change it as needed to obtain the credentials:"